src/components/shared/change-password-form.tsx

```
@@ -0,0 +1,140 @@
+import { toast } from "sonner";
+import { useChangePassword } from "@/hooks/profile/use-change-password.hook";
+import { FormInput } from "@lib/types/profile/change-password";
+import { changePassword } from "@lib/actions/profile/change-password.action";
+import { ChangePasswordFormSchema, ChangePasswordValues } from "@lib/schemas/profile/change-password.schema";
+
+export default function ChangePasswordForm() {
+  // Translation
+  const t = useTranslations("change-password");
+
+  // Hooks
+  const { mutate, isPending } = useChangePassword();
```

ESLint

[error] 'mutate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)